src/components/common.js

```diff
@@ -25,6 +25,26 @@ const styles = {
     display: 'inline-flex', justifyContent: 'space-around', alignItems: 'center', height: '2.25rem',
     fontWeight: 500, fontSize: 14, textTransform: 'uppercase', whiteSpace: 'nowrap',
     userSelect: 'none'
+  },
+  tabBar: {
+    container: {
+      display: 'flex', alignItems: 'center',
+      fontWeight: 400, textTransform: 'uppercase',
+      height: '3.75rem',
```

Comment:
This seems a little too tall. Maybe try `2.25rem` to match the height of our buttons?